src/components/Login.jsx

```diff
@@ -3,19 +3,67 @@ import { useHistory } from "react-router-dom";
 import { UserContext } from "./Context";
 import { useGoogleLogin } from "react-google-login";
 import GoogleLogo from "../assets/images/google-logo.svg";
+import axios from "axios";
 
 function Login() {
   const userContext = useContext(UserContext);
   const history = useHistory();
 
+
   const onSuccess = (res) => {
     console.log("Login Success", res.profileObj);
-    userContext.setUserData({
-      token: true,
-      imageUrl: res.profileObj.imageUrl,
-      name: res.profileObj.name,
-    });
+    
     history.push("/");
+
+    const fetchLogin = async () => {
+  
+      const result = await axios.get("http://localhost:8080/api/users/login/"+res.profileObj.googleId)
```

Comment:
Coś jest nie tak z tą funkcją (gdy użytkownika jeszcze nie ma w bazie). Jak loguję się to dostaję błąd.
![image](https://user-images.githubusercontent.com/25390589/98484144-b619de80-220d-11eb-9a36-84c781e397b7.png)
Po odświeżeniu i ponownej próbie zalogowania, wszystko przechodzi poprawnie i użytkownik zostaje zalogowany.